olive/passes/onnx/common.py

```diff
@@ -183,3 +185,29 @@ def model_proto_to_olive_model(
         onnx.checker.check_model(olive_model.model_path)
 
     return olive_model
+
+
+def get_lora_name_patterns() -> List[str]:
+    """Get the node name patterns for lora modules."""
+    return [
+        f".*[./]{name}[./]{matmul}$"
+        for name in ["default", "default_1", "lora_A", "lora_B"]
+        for matmul in ["MatMul", "MatMul_Q4"]
+    ]
```

Comment:
nitpick: This can be a constant set rather than a function.

Reply:
good idea. done!